introduction.tex

```diff
@@ -1,35 +1,58 @@
 \section{Introduction}
 This paper uses CYCLUS, the agent-based simulator \cite{huff_fundamental_2016} to analyze
-the future nuclear inventory in the European Union. This paper focuses on different scenarios of France,
-and how potential French legislation and policy can change the nuclear landscape of the European
-Union. This paper begins with analysis of a once through fuel cycle, where all the spent nuclear fuel,
-goes to a central repository after discharge from reactors. Next, a reprocessing fuel cycle is evaluated.
-In this fuel cycle only French \gls{PWR}s accept \gls{MOX} fuel, where the spent \gls{UOX} from
-all the member states is recycled in France. Finally, a \gls{LWR} to \gls{SFR} transition scenario \cite{wigeland_nuclear_2014}
-is modeled,
-where a once-through-cycle transitions to a closed cycle with continuous reprocessing of Plutonium and minor actinides.
+the future nuclear inventory in the European Union. This paper focuses on the spent fuel
+inventory in the EU nations in 2050, and analyzes various potential strategies of spent fuel
+management.
+A major focus of this paper is to see if France has an incentive
+to receive all the \gls{SNF} from EU nations to create \gls{MOX}.
+The \gls{MOX} created will fuel French transition to a \gls{SFR} fleet
+with the goal of not having to build new \gls{LWR}s.
+A CYCLUS simulation is run to see
+how much spent fuel is created from different nations from 1970 to 2050, and then amount
+of \gls{MOX} that can be created with the \gls{SNF} inventory is calculated.
+The paper assumes a once-through cycle for all the 
+EU nations with the exception of France. France can reprocess spent \gls{UOX} and \gls{MOX} to
+make \gls{MOX} from reprocessed plutonium and depleted uranium (tails). It is assumed that
+\gls{MOX} is reprocessed infinitely. 
+After obtaining the \gls{SNF} inventory of all EU in 2050, a separate
```

Comment:
I do feel like it is worth mentioning that CYCLUS can run all these things at once, 
but I think it is easier for the reader
to view the results if the simulations were done separately,
since the first simulation is all of EU and the latter just France.
The timescale is also different, which might confuse the reader.